registration/registration-client/src/main/java/io/mosip/registration/controller/SettingsController.java

```diff
@@ -199,7 +199,9 @@ private void loadFXML(String fxmlName, String headerLabel) {
 		try {
 			exitWindow();
 			if (getRegistrationDTOFromSession() != null) {
-				goToSettingsFromRegistration();
+				if (!goToSettingsFromRegistration()) {

```

Comment:
Can you please reduce this nested if block to single if block @HimajaDhanyamraju2

Reply:
Modified the nested block to single block.